### DISCUSSION TITLE
Latest Version Not working with nextjs

### DISCUSSION THREAD

**Discussion by collinpowell** [system-ui/theme-ui#2140] [Q&A] [answered]
**Describe the bug**
I get an error message when i use them-ui 0.13.1 on nextjs frame work

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by lachlanjc** [comment, marked as answer]
Hi! Please provide a link to a reproduction & more details on how you're using Theme UI. Next.js is most certainly supported, & it is working with the latest Theme UI, but if you're running into a specific issue we're happy to assist.
